csrc/sparse.h

```diff
@@ -2,69 +2,82 @@
 
 #include <torch/library.h>
 
-int64_t cuda_version();
+#ifdef _WIN32
+#if defined(torchsparse_EXPORTS)
```

Comment:
CMake automatically adds this definition when building a shared library https://cmake.org/cmake/help/v3.0/prop_tgt/DEFINE_SYMBOL.html